src/components/ModalContent/index.js

```diff
@@ -0,0 +1,46 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import cn from 'classnames'
+
+
+export default class ModalContent extends PureComponent {
+  static propTypes = {
+    children: PropTypes.oneOfType([
+      PropTypes.node,
+      PropTypes.arrayOf(PropTypes.node),
+    ]),
+    className: PropTypes.string,
+  }
+
+  constructor (props) {
```

Comment:
you don't need to use the constructor.

```javascript
content = React.createRef()
```

Will work

Reply:
Dope, I dig that a lot more.  Thanks!